testutil/sample/launch.go

```diff
@@ -136,3 +138,9 @@ func MsgEditChain(
 		initialGenesis,
 	)
 }
+
+// GenesisHash returns a sample sha256 hash of custom genesis for GenesisURL
+func GenesisHash() string {
```

Comment:
If we merge the #230 first, we can reuse this function:

https://github.com/tendermint/spn/pull/230/files#diff-6619c7d1a9a468c88892c0e3dcd2961584eae593aae45d7071bc902b9eb59896R13-R18